reana_commons/utils.py

```diff
@@ -181,9 +181,13 @@ def build_caching_info_message(job_spec,
     return caching_info_message
 
 
-def get_workspace_disk_usage(workspace, summarize=False):
+def get_workspace_disk_usage(workspace, summarize=False, block_size=None):
     """Retrieve disk usage information of a workspace."""
-    command = ['du', '-h']
+    command = ['du']
+    if block_size:
+        command.append('-{}'.format(block_size))
```

Comment:
Perhaps put an explicit check if `-b` then accept, if `-k` then accept, otherwise `-h`, just to be safe... We could also have a check on the REST API server side, but an if/else tree here will prevent us from shooting ourselves in the foot as well, just in case